### ISSUE TITLE
Breaking change in casbin v2.55.0+

### ISSUE THREAD

**Issue by zredinger-ccc** [pckhoi/casbin-pgx-adapter#6]
https://github.com/pckhoi/casbin-pgx-adapter/blob/4e6df0dc63d2547b1afe1940bd1e6d3453b59b0e/adapter.go#L377

It appears this is a breaking change for the adapter.

**Comment by pckhoi** [comment]
I'm currently busy so I won't be able to address this anytime soon. Feel free to contribute of course.

**Comment by pckhoi** [comment]
Also just so that we are on the same page, please add the error that you see in the issue description.

**Comment by edwardzjl** [comment]
Casbin [v2.56.0](https://github.com/casbin/casbin/tree/v2.56.0) add an error return to their `persist.LoadPolicyLine` function, which will break [this line](https://github.com/pckhoi/casbin-pgx-adapter/blob/5f6296d2baafd7c2f6abe923c9ce0598c472b667/adapter.go#L377):

``` go
func (a *Adapter) LoadFilteredPolicy(model model.Model, filter interface{}) error {
    ...
    err := a.loadFilteredPolicy(model, filterValue, persist.LoadPolicyLine)
    ...
}
```

See their [issue](https://github.com/casbin/casbin/issues/1083#issuecomment-1221529782) and [PR](https://github.com/casbin/casbin/pull/1085)

I've made a [PR](https://github.com/pckhoi/casbin-pgx-adapter/pull/7) to fix this problem

**Comment by pckhoi** [comment]
Thanks @edwardzjl! CI failed to make an automatic release but your changes should be available in at version `v2.2.0`.
